### PR TITLE
1st Batch of programs: errorReporterRefactor project

### PR DESCRIPTION
**Pull request by jrosengarden**
There IS method to my madness;  I'm working thru the CPP files alphabetically...for lack of a better methodology!

### CODE REVIEW

File: guiclient/accountNumber.cpp [hunk 1]
```diff
@@ -346,10 +348,10 @@ void accountNumber::populateSubTypes()
     sub.bindValue(":subaccnttype_accnt_type", "Q");
   sub.exec();
   _subType->populate(sub);
-  if (sub.lastError().type() != QSqlError::NoError)
+  if (ErrorReporter::error(QtCriticalMsg, this, tr("Error Retreiving SubTypes"),
```
Comment (gilmoskowitz) [inline guiclient/accountNumber.cpp]: typo - retr**ei**ving

**Comment by gilmoskowitz** [comment]
Nice.
1 question - is there an editor setting that's changing the indentation of some lines? Most `return` statements in the change zone are getting pushed 2 chars further to the right but are otherwise unchanged.

**Comment by jrosengarden** [comment]
Gil:

1.  Should I correct that typo then re-push and make a new Pull request?

2.  I’ll look at the editor and see if there is something going on.  Sorry about that.

**Comment by gilmoskowitz** [comment]
Jeff,

Just fix the typo and push. This pull request will update itself, then I'll merge it.

Re: the editor settings, don't worry about changing this pull request. Just see if it's something simple to adjust for the next round.

Thanks much.
Gil

**Comment by jrosengarden** [comment]
Gil:

Typo corrected & pushed.

Please accept my apologies for such a stupid mistake.  I SHOULD NOT have let that get thru.  I am embarrassed!

**Comment by jrosengarden** [comment]
Gil:

Also found the problem with the extra spaces on the “return” lines.  Also my stupid fault.  Fixed.  Once again…my apologies.  I’m supposed to be helping here…not causing new problems.

**Comment by gilmoskowitz** [comment]
Jeff,

No problem. It always takes a while to get set up for a new dev environment, new coding conventions, etc.

Gil

**Pull request merged**